src/components/DotNavigation/DotNavigation.tsx

```diff
@@ -3,7 +3,8 @@ import styles from './DotNavigation.st.css';
 import { RadioButton, RadioButtonKeyDownEvent } from 'wix-ui-core/radio-button';
 import { DotNavigationDataKeys, DotNavigationDataHooks } from './dataHooks';
 import classNames from 'classnames';
-import * as _ from 'lodash';
+import isNaN from 'lodash/isNaN';
+import isNumber from 'lodash/isNumber';
```

Comment:
@zhukovka 
Even though your solution is probably the correct one, i'd rather we do the `tsconfig` change in a different, and controlled PR.
So for now, maybe instead write:
```suggestion

const isNan = require('lodash/isNan');
const isNumber = require('lodash/isNumber');

```

Reply:
ok